visualization/app/codeCharta/model/customConfig/customConfig.api.model.ts

```diff
@@ -5,9 +5,28 @@ export enum CustomConfigMapSelectionMode {
 	MULTIPLE = "MULTIPLE",
 	DELTA = "DELTA"
 }
+
 export interface CustomConfig {
 	id: string
 	name: string
+	creationTime: number
+	mapSelectionMode: CustomConfigMapSelectionMode
+	assignedMaps: string[]
+	mapChecksum: string
+	customConfigVersion: string
+
+	stateSettings: {
+		appSettings: AppSettings
+		dynamicSettings: DynamicSettings
+		fileSettings: FileSettings
+		treeMap: TreeMapSettings
+	}
+}
+
+export interface ExportCustomConfig {
```

Comment:
Why do we need this code duplication? We could type alias it, if the name provides same value like `type ExportCustomConfig = CustomConfig`. Does the extra name increase readability / maintainability?

Reply:
The reason for that is to achive decoupling between CustomConfig and ExportCustomConfig. We have introduced a separate API Versions for ExportCustomConfig as well. A change of CustomConfig must not affect ExportCustomConfig.

Reply:
Can you give me an example where CustomConfig changes and ExportCustomConfig doesn't or vice versa?

Reply:
Thanks for triggering my brain. This decoupling is indeed not needed because CustomConfig and ExportCustomConfig should always change together. Let's keep the alias `ExportCustomConfig` to keep the opportunity to add properties dedicated to the download for each CustomConfig.